modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -156,6 +156,45 @@ public RustClientCodegen() {
         return postProcessModelsEnum(objs);
     }
 
+    @SuppressWarnings({"static-method", "unchecked"})
+    public Map<String, Object> postProcessAllModels(Map<String, Object> objs) {
+        // Index all CodegenModels by model name.
+        Map<String, CodegenModel> allModels = new HashMap<>();
+        for (Map.Entry<String, Object> entry : objs.entrySet()) {
+            String modelName = toModelName(entry.getKey());
+            Map<String, Object> inner = (Map<String, Object>) entry.getValue();
+            List<Map<String, Object>> models = (List<Map<String, Object>>) inner.get("models");
+            for (Map<String, Object> mo : models) {
+                CodegenModel cm = (CodegenModel) mo.get("model");
+                allModels.put(modelName, cm);
```

Comment:
It looks as if we have only one `modelName` per entry. Are we expecting the list returned by `inner.get("models")` to have only one member?

Reply:
I shamelessly stole it from another generator (I think `haskell`) but since the model gives you an `ArrayList`, I expect that it's possible in some cases that there are multiple elements.

Reply:
OK. We can probably assume the `haskell` authors knew what they were doing, then.